test.sh

```diff
@@ -524,7 +524,8 @@ sub=$(curl -sSfg -u $SK: $HOST/v1/subscriptions/$sub_id \
            -d items[0][id]=$sub_item_id)
 
 in=$(curl -sSfg -u $SK: $HOST/v1/invoices \
-          -d customer=$cus)
+          -d customer=$cus \
+          -d pending_invoice_items_behavior=include)
```

Comment:
If this new option is not really implemented, I'd prefer not using it in tests (I believe that's what we do for other unimplemented options).

Reply:
OK, I removed it!